src/sentry/incidents/subscription_processor.py

```diff
@@ -573,7 +576,7 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                             self.trigger_alert_counts[trigger.id] = 0
 
                         if (
-                            not self.has_anomaly(potential_anomaly, trigger.label)
+                            not self.has_anomaly(potential_anomaly, trigger.label, fake_anomalies)
```

Comment:
@ceorourke why not just just do an OR here:
```
self.has_anomaly(potential_anomaly, trigger.label) or fake_anomalies
```
Seems a bit strange to pass it to this function and have it bypass the the whole function